capture-traffic/capture-from-ios.md

```diff
@@ -12,70 +12,102 @@ previous_url: /knowledge-base/configure-ios, /get-started/mobile-traffic/configu
 
 This article describes using Fiddler Everywhere to capture HTTPS traffic from iOS devices and simulators.
 
-To capture HTTPS traffic on iOS devices or simulators, perform the following steps:
-
-1. [Provide the prerequisites](#prerequisites).
-1. [Configure Fiddler Everywhere](#configuring-fiddler-everywhere-host).
-1. [Configure the iOS device](#configuring-ios-devices).
-1. [Configure the iOS simulator](#configuring-ios-simulator).
-1. [Capture mobile traffic](#capture-traffic).
-
 ## Prerequisites
 
 - Install [the latest version of Fiddler Everywhere](https://www.telerik.com/download/fiddler-everywhere).
 
 - Use an iOS device or an iOS simulator within the same local network as the Fiddler Everywhere host.
 
-## Configuring Fiddler Everywhere Host
+>important It's common for many company networks to be restricted by security tooling or administrative policies. To capture remote traffic, ensure that your system administrators are not limiting the network discovery and usage, or consider using a public hotspot for your Fiddler host and remote device.
+
+
+## Capturing Remote iOS Traffic (Automated)
+
+Fiddler Everywhere provides an automated guide to configure remote iOS capturing through the following steps:
+
+1. Start Fiddler Everywhere. 
+
+1. Open the **Home** pane.
+
+1. Open the **Remote Devices** screen.
+
+1. Open the **iOS** tutorial and follow the instructions.
+
+The tutorial requires administrative privileges for the current user (on the iOS device) to install Fiddler's CA (certificate authority) and to set manual proxy. Once the steps are executed, you can immediately capture HTTPS traffic from the remote iOS device.
```

Comment:
Same comment as for Android - when introducing abbreviation, use the full name first:
Fiddler's CA (certificate authority) -> Fiddler's Certificate Authority (CA)